testing/percona-xtradb-cluster-galera/APKBUILD

```diff
@@ -0,0 +1,60 @@
+# Contributor: Peter Szalatnay <theotherland@gmail.com>
+# Maintainer: Peter Szalatnay <theotherland@gmail.com>
+pkgname=percona-xtradb-cluster-galera
+pkgver=5.7.16
+_pkgver=5.7.16-27.19
```

Comment:
i dont like this. we have a script called `abump` that we use for simple version bumps. if we run `abump percona-xtradb-cluster-galera-5.7.17` to upgrade, then would it modify the `pkgver` only, build would succeed but the actual downloaded and built version would be 5.7.16 since that was unmodified.

better do:
````sh
_pkgver="$pkgver-27.19"
````

The would alteast the download fail and we'd not end up with a package with wrong version number.